users/config.rst

```diff
@@ -48,22 +50,27 @@ located:
 
 The database is stored either in the same directory as the config (usually the
 default), but may also be located in one of the following directories (Unix-like
-platforms only):
+platforms only) for legacy reasons:
 
-* If a database exists in the old default location, that location is
-  still used.
-* If ``$XDG_DATA_HOME`` is set, use ``$XDG_DATA_HOME/syncthing``.
-* If ``~/.local/share/syncthing`` exists, use that location.
-* Use the old default location (same as config).
+* ``$XDG_DATA_HOME/syncthing``
+* ``~/.local/share/syncthing``
+* Any location specified by the ``--data`` or ``--home`` flags or the
+  corresponding environment varibles (``$STDATADIR`` or ``STHOMEDIR``).
 
-The location of the database can be changed using the ``--data`` flag. The
-``--home`` flag sets both config and database locations at the same time.
-The database contains the following files:
+The database directory contains the following files, among others:
 
 :file:`index-{*}.db`
     A directory holding the database with metadata and hashes of the files
     currently on disk and available from peers.
 
+:file:`syncthing.log`
```

Comment:
`csrftokens.txt` also lands here if I read the code correctly.  Move from the paragraph above please.